recipe/meta.yaml

```diff
@@ -20,6 +20,15 @@ requirements:
     - {{ stdlib("c") }}
     - cmake
     - make
+    - python
+    - cython
+  host:
+    - python
+    - cython
+    - setuptools
+    - pip
```

Comment:
In addition to `cython`, why are `setuptools` and `pip` needed as [`host` requirements](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#host)?

Reply:
Those dependencies where installed in attempt to fix issues while trying to build multiple-output package and can be removed for now. 
I think that switching to packaging another tools or working in parallel on both `hepmc2` and `LHAPDF` (or other tool)  will be a good idea as multiple-output package takes way more time than I expected for some reason.